docs/aws-professional/services.md

```diff
@@ -22,168 +22,168 @@ For an overview of Azure for AWS users, see [Introduction to Azure for AWS exper
+| Content delivery network      | CloudFront             | [Content Delivery Network](https://azure.microsoft.com/services/cdn/)                                                                                 | Global content delivery network that transfers audio, video, applications, images, and other files.                           |
+| Dedicated network             | Direct Connect         | [ExpressRoute](https://azure.microsoft.com/services/expressroute/)                                                                                          | Establishes a dedicated, private network connection from a location to the cloud provider.                         |
+| Load balancing                | Elastic Load Balancing | [Load Balancer](https://azure.microsoft.com/services/load-balancer/) <br/><br/>[Application Gateway](https://azure.microsoft.com/services/application-gateway/) | Automatically distributes incoming application traffic to add scale, handle failover, and route to a collection of resources.                                                                                                                       |
 
 
 ## Database
 
-| Area                    | AWS Service                          | Azure Service                                                                                                         | Description                                                                                                                                             |
+| Service                    | AWS                          | Azure                                                                                                         | Description                                                                                                                                             |
 |-------------------------|--------------------------------------|-----------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Relational database     | RDS                                  | [SQL Database](https://azure.microsoft.com/services/sql-database/)                                       | Relational database-as-a-service (DBaaS) where the database resilience, scale, and maintenance are primarily handled by the platform.                   |
+| Relational database     | RDS                                  | [SQL Database including Postgres and MySQL](https://azure.microsoft.com/services/sql-database/)                                       | SQL Database is a high-performance, reliable, and secure database you can use to build data-driven applications and websites, without needing to manage infrastructure.                 |
```

Comment:
Provide separate links for Postgres and MySQL?

Reply:
I think separate links may help here. The key is that folks don't know we have support for those additional two dbs.